website/docs/d/cloudant_database.html.markdown

```diff
@@ -0,0 +1,68 @@
+---
+layout: "ibm"
+page_title: "IBM : cloudant_database"
+description: |-
+  Get information about cloudant_database
+subcategory: "Cloudant"
```

Comment:
Already the ibm_cloudant is under "Cloud Databases" should we use the same category
<img width="418" alt="Screenshot 2022-03-15 at 5 08 15 PM" src="https://user-images.githubusercontent.com/24930306/158370052-06b21c51-be10-48f4-9ea2-a81c1458299b.png">

Should we use the same or seperate all cloudant to new category ...?

If same then use the same subcategory ""Cloud Databases"  
If new then move the existing cloudant one also under new sub Category and add a entry in /website/allowed-subcategories.txt file